bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_2jets/Wgamma_Monophoton_Ptg130_2jets_run_card.dat

```diff
@@ -0,0 +1,251 @@
+  .false.	=  gridpack   !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  3000	=  nevents  ! Number of unweighted events requested 
+  0	=  iseed    ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  1	=  lpp1     ! beam 1 type 
+  1	=  lpp2     ! beam 2 type
+  6500	=  ebeam1   ! beam 1 total energy in GeV
+  6500	=  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0	=  polbeam1  ! beam polarization for beam 1
+  0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  'lhapdf'	=  pdlabel      ! PDF set                                     
+   $DEFAULT_PDF_SETS  	=  lhaid      ! if pdlabel=lhapdf, this is the lhapdf number
+   $DEFAULT_PDF_MEMBERS = reweight_PDF
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+  F	=  fixed_ren_scale   ! if .true. use fixed ren scale
+  F	=  fixed_fac_scale   ! if .true. use fixed fac scale
+  91.1880	=  scale             ! fixed ren scale
+  91.1880	=  dsqrt_q2fact1     ! fixed fact scale for pdf1
+  91.1880	=  dsqrt_q2fact2     ! fixed fact scale for pdf2
+  1	=  scalefact         ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+  1	=  ickkw             ! 0 no matching, 1 MLM, 2 CKKW matching
+  1	=  highestmult       ! for ickkw=2, highest mult group
+  1	=  ktscheme          ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+  1	=  alpsfact          ! scale factor for QCD emission vx
+  F	=  chcluster         ! cluster only according to channel diag
+  T	=  pdfwgt            ! for ickkw=1, perform pdf reweighting
```

Comment:
We recommend pdfwgt = F due to know MC/data mismodeling. Is your choice on purpose?